src/pages/billing/Project.js

```diff
@@ -26,8 +26,16 @@ import { billingRoles } from 'src/pages/billing/List'
 const workspaceLastModifiedWidth = 150
 const workspaceExpandIconSize = 20
 
+const workspaceBillingStatusIcon = (() => {
+  const size = 16
+  const blank = div({ style: { width: size } },
+    [div({ className: 'sr-only' }, ['Status'])])
+  return shape => shape ? icon(shape, { size }) : blank
+})()
+
 const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort, onSort }) => {
   return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem', marginBottom: '0.5rem' } }, [
+    workspaceBillingStatusIcon(null),
```

Comment:
Passing `null` implies that this takes an argument. Better to pass nothing.

Reply:
It does take an argument though?

Reply:
I had to look more carefully at the definition. Indeed I was wrong.